Rubberduck.Core/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -529,7 +621,7 @@ private void ExecuteCopyResultsCommand(object parameter)
 
         private bool CanExecuteCopyResultsCommand(object parameter)
         {
-            return !IsBusy && _results != null && _results.Any();
+            return !IsBusy && _resultsAll != null && _resultsAll.Any();
```

Comment:
Hmm, so copying will copy all results, irrespective of the filters I set up? Seems counterintuitive.

Reply:
How should this be addressed? I don't know what this does. Should I create a separate backing field `_filteredResults` and use that instead of `_resultsAll`? Thinking about the name `_allResults` seems a bit better.